Cargo.toml

```diff
@@ -8,6 +8,7 @@ members = [
   "integration_test",
   "file_dbg_macros",
   "win_setupgui",
+  "shared_sql_utils",
```

Comment:
I had to extract some files to a new crate to avoid triangular symbol resolution. Essentially, the setup dll *cannot* include anything from the *odbc crates because the symbols for the new function I introduced to test the connection credentials collide.